packages/portal/src/app/rentalpage/rentalpage.component.ts

```diff
@@ -1,6 +1,7 @@
 import { CommonModule } from "@angular/common";
 import { Component, OnInit, inject, signal } from "@angular/core";
 import { ActivatedRoute, Navigation, Router } from "@angular/router";
+import {MatMenuModule} from '@angular/material/menu';
```

Comment:
To comply with project conventions, please add a space between curly brackets and import names. Thank you!

Reply:
@anfibiacreativa done, thank you for reviewing!